core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectoryIdGenerator.java

```diff
@@ -41,7 +42,20 @@ public InodeDirectoryIdGenerator(ContainerIdGenerable containerIdGenerator) {
     mContainerIdGenerator = Preconditions.checkNotNull(containerIdGenerator);
   }
 
+  /**
+   * @return the next directory id
+   */
   synchronized long getNewDirectoryId() {
+    return getNewDirectoryId(null);
```

Comment:
nit: if you pass `NoopJournalContext.INSTANCE` than the callee can assume the journalContext is never null